src/plugins/git/types.js

```diff
@@ -7,4 +7,6 @@ export type Hash = string;
 export type Commit = {|
   +hash: Hash,
   +parentHashes: $ReadOnlyArray<Hash>,
+  +shortHash: string, // short unambiguous identifier
```

Comment:
I take issue with the use of _unambiguous_ here, because it means that `mergeRepository` is no longer correct. What should we do about this?